### DISCUSSION TITLE
Lights being in a group effect their last_seen value?

### DISCUSSION THREAD

**Discussion by reefland** [Koenkk/zigbee2mqtt#10551] [General]
All of the devices I have added to various zigbee light group currently show a `last_seen` on a scale of days, yet they operate fine. Z2M has no problem interacting with them via the group.  But the `last_seen` does not get updated.

Is their a behavior differences with devices being added to a zigbee group that would stop them from sending out status packet once and a while? Mains power devices tend to check-in regularly. 

9 of my 10 Hue devices are in light groups.  (2 in a group for Blooms, 4 in a group for light strips, 3 in a group for Hue Play devices).   All of these have not been "seen" for days, yet all operate via Z2M instantly when requested to and are part of Home Assistant automations which process perfectly throughout the day.

The one Hue device I have not added to a group updates its `last_seen` every 5 to 8 minutes.  

I recently added 2 IKEA bulbs to a zigbee light group and they have not updated  `last_seen` in more than 24 hours now.

**Comment by Koenkk** [comment]
Groups are broadcast commands, so nothing get send back to the coordinator. If you control them individually an acknowledge is send on each command and the last seen is updated.

**Reply by NikitaKorneev** [reply]
I have an issue of tuya sporlights not "acknowledging" the command that I sent but executing it fine. Is there way to force them to do so?

**Comment by reefland** [comment]
But don't mains powered devices such as bulbs which act as routers still send out a status packet every few minutes?  Does this behavior stop when added to a zigbee group?

**Reply by Koenkk** [reply]
> But don't mains powered devices such as bulbs which act as routers still send out a status packet every few minutes?

They don't

**Reply by reefland** [reply]
Good to know, thank you.

**Reply by NikitaKorneev** [reply]
> > But don't mains powered devices such as bulbs which act as routers still send out a status packet every few minutes?
> 
> 
> 
> They don't

Is there a way to force them to do so?
